src/util/deserialize.ts

```diff
@@ -95,32 +100,32 @@ class Deserializer {
     // so we don't double-process the same thing
     // must push before relationships
     this._deserialized.push(instance);
-    this._processRelationships(instance, datum.relationships, includeDirective);
+    this._processRelationships(instance, datum.relationships || {}, includeDirective);
 
     // remove objects marked for destruction
     this._removeDeletions(instance, includeDirective);
 
     // came from server, must be persisted
-    instance.isPersisted(true);
+    instance.isPersisted = true
 
     return instance;
   }
 
-  _removeDeletions(model: Model, includeDirective: Object) {
+  _removeDeletions(model: JSORMBase, includeDirective: Object) {
     Object.keys(includeDirective).forEach((key) => {
       let relatedObjects = model[key];
       if (relatedObjects) {
         if (Array.isArray(relatedObjects)) {
           relatedObjects.forEach((relatedObject, index) => {
-            if (relatedObject.isMarkedForDestruction() || relatedObject.isMarkedForDisassociation()) {
+            if (relatedObject.isMarkedForDestruction || relatedObject.isMarkedForDisassociation) {
```

Comment:
I support this, but we should keep a list of all the backwards incompatibilities introduced (even with 1.0, helpful to update the docs if nothing else)

Reply:
Yeah, I haven't pushed it yet, but I'm maintaining a test file with all failing specs documenting backwards-breaking changes.